docs/Toy/toy/compiler.py

```diff
@@ -83,4 +96,22 @@ def transform(
     if target == "scf":
         return
 
+    # When the commented passes are uncommented, we can print RISC-V assembly
+
+    SetupRiscvPass().apply(ctx, module_op)
+    # LowerFuncToRiscvFunc().apply(ctx, module_op)
+    # LowerToyAccelerator().apply(ctx, module_op)
+    # LowerMemrefToRiscv().apply(ctx, module_op)
+    # LowerPrintfRiscvPass().apply(ctx, module_op)
+    # LowerArithRiscvPass().apply(ctx, module_op)
+    DeadCodeElimination().apply(ctx, module_op)
+    # ReconcileUnrealizedCastsPass().apply(ctx, module_op)
+
+    DeadCodeElimination().apply(ctx, module_op)
```

Comment:
twice?

Reply:
Ah yes I guess not really necessary right now, I'll remove it in the next PR